bokun.yaml

```diff
@@ -5141,118 +5142,179 @@ definitions:
         format: double
   ActivityAvailabilityDto:
     properties:
+      id:
+        type: string
+        description: 'Value "startTimeId_YYYYMMDD" (YYYYMMDD - current request date).'
```

Comment:
I don't understand what you mean by "current request date"

The date in the id is the date of the departure this particular availability is for.

Reply:
Meaning currentDate() for the moment, when the request is sent.

Reply:
I see, thank you for identifying!